org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui/src-gen/org/eclipse/xtext/example/domainmodel/ui/AbstractDomainmodelUiModule.java

```diff
@@ -208,6 +212,7 @@ public void configureIResourceDescriptionsPersisted(Binder binder) {
 	}
 	
 	// contributed by org.eclipse.xtext.xtext.generator.generator.GeneratorFragment2
+	@Override
```

Comment:
Edited manually?

Reply:
will revert all additions of override that were added